fiftyone/server/routes/geo.py

```
@@ -0,0 +1,76 @@
+import fiftyone.core.odm as foo
+import fiftyone.core.view as fov
+import fiftyone.server.view as fosv
+from fiftyone.server.decorators import route
+
+
+class GeoPoints(HTTPEndpoint):
+    @route
+    async def post(self, request: Request, data: dict) -> dict:
+        dataset = data.get("dataset")
+        path = data.get("path")
+        filters = data.get("filters", None)
```

Ruff (0.8.2)

Use `data.get("filters")` instead of `data.get("filters", None)`

Replace `data.get("filters", None)` with `data.get("filters")`

(SIM910)